### PR TITLE
bugfix for issue #2491

### PR DESCRIPTION
**Pull request by OfirOshir**
Thanks for submitting a PR, your contribution is really appreciated!

Here's a quick checklist that should be present in PRs:

- [x] Add a new news fragment into the changelog folder
  * name it `$issue_id.$type` for example (588.bug)
  * if you don't have an issue_id change it to the pr id after creating the pr
  * ensure type is one of `removal`, `feature`, `bugfix`, `vendor`, `doc` or `trivial`
  * Make sure to use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files."
- [x] Target: for `bugfix`, `vendor`, `doc` or `trivial` fixes, target `master`; for removals or features target `features`;
- [x] Make sure to include reasonable tests for your change if necessary

Unless your change is a trivial or a documentation fix (e.g.,  a typo or reword of a small section) please:

- [x] Add yourself to `AUTHORS`;

### CODE REVIEW

**Comment by OfirOshir** [comment]
the tests failed but i don't understand why can someone plz explain to me?

**Comment by obestwalter** [comment]
> python3.5: command not found\\n\\nThe `python3.5\' command exists in these Python versions:\\n  3.5\\n  3.5.3\\n\\n"'

Looks like a glitch on travis - nothing to do with your changes.

**Comment by nicoddemus** [comment]
Indeed this is fixed by #2790

**Comment by OfirOshir** [comment]
ok, so what can i do?

**Comment by obestwalter** [comment]
You can close and reopen the pr then the tests are triggered again.

**Comment by nicoddemus** [comment]
Actually #2790 has not been merged yet. @OfirOshir please hold on until we merge it, thanks for your patience.

File: changelog/2491.bugfix [hunk 1]
```diff
@@ -0,0 +1,4 @@
+If an exception happens while loading a plugin, PyTest no longer hides the original traceback.
```
Comment (nicoddemus) [inline changelog/2491.bugfix]: It is actually `pytest`, not `PyTest`. 😉

**Comment by obestwalter** [comment]
ups ... sorry 😊

**Comment by nicoddemus** [comment]
No worries! :smile:

**Comment by RonnyPfannschmidt** [comment]
thanks 👍

File: _pytest/config.py [hunk 2]
```diff
@@ -423,12 +423,12 @@ def import_plugin(self, modname):
...
-            for attr in ('name', 'path'):
-                if hasattr(e, attr):
-                    setattr(new_exc, attr, getattr(e, attr))
-            raise new_exc
+            new_exc_type = ImportError
+            new_exc_message = 'Error importing plugin "%s": %s' % (modname, safe_str(e.args[0]))
+            new_exc = new_exc_type(new_exc_message)
+
+            six.reraise(new_exc_type, new_exc, sys.exc_info()[2])
+
```
Comment (srinivasreddy) [inline _pytest/config.py]: why this verbose? Why not do it this way?
```
six.reraise(ImportError, ImportError('Error importing plugin "%s": %s' % (modname, safe_str(e.args[0]))), sys.exc_info()[2])
```

Comment (nicoddemus) [reply _pytest/config.py]: Well both versions are correct, it is more of a matter of style in this case I guess.

**Pull request merged**